specs/plugin-extensibility.md

```diff
@@ -22,13 +22,28 @@ Keys and associated certificates used for signing artifacts using Notation could
 
 Notation will invoke plugins as executable, pass parameters using command line arguments, and use standard IO streams to pass request/response payloads. This mechanism is used as Go language (used to develop [Notation Go library](https://github.com/notaryproject/notation-go)) does not have a [in built support](https://github.com/golang/go/issues/19282) to load and execute plugins that works across OS platforms. Other mechanisms like gRPC require every plugin to be implemented as a service/daemon.
 
+### Package and distribute plugin file
+
+To ensure plugin installation is compatible with Notation, plugin publisher SHOULD follow these conventions to package and distribute plugin files:
```

Comment:
We don't need to mention `plugin installation with Notation` here. Because the main purpose of this new content is for `plugin package and release`. How to install the plugin is not decided by the `plugin publisher`.
```suggestion
Plugin publishers SHOULD follow these conventions to package and release plugins:
```